### DISCUSSION TITLE
cell - Mitosis / Meiosis

### DISCUSSION THREAD

**Discussion by nphias** [holochain/holochain#2993] [Ideas]
so i was wondering why cells dont have the power clone themselves..  only the agent-ws-client can call createClone.. what if you wanted some interaction cell logic that resulted in cell cloning itself to a new "daughter cell" with different properties. 

Also a cell creating a different deferred cell (role) from the appmanifest.. at the time that it is needed.  that would be nice

**Comment by mattyg** [comment]
+1 -- I would like to see all functions currently exposed to the app api available from the wasm host for coordinators -- create clone, disable clone, enable clone, and network info
